framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -981,60 +1068,76 @@ protected boolean visitIntersectionSubtype(
    * @param supertype a type that might be a supertype
    * @return true if {@code subtype} is a subtype of {@code supertype}
    */
-  protected boolean visitTypevarSupertype(
+  protected boolean visitType_Typevar(
       AnnotatedTypeMirror subtype, AnnotatedTypeVariable supertype) {
     return isSubtypeCaching(subtype, supertype.getLowerBound());
   }
 
   /**
-   * A type variable is a subtype if its upper bounds is below the supertype. Note: When comparing
-   * two type variables this method and visitTypevarSupertype will combine to isValid the subtypes
+   * A type variable is a subtype if its upper bound is below the supertype. Note: When comparing
+   * two type variables, this method and visitType_Typevar will combine to isValid the subtypes
    * upper bound against the supertypes lower bound.
    */
-  protected boolean visitTypevarSubtype(
+  protected boolean visitTypevar_Type(
       AnnotatedTypeVariable subtype, AnnotatedTypeMirror supertype) {
-    AnnotatedTypeMirror upperBound = subtype.getUpperBound();
-    if (TypesUtils.isBoxedPrimitive(upperBound.getUnderlyingType())
+    AnnotatedTypeMirror subtypeUpperBound = subtype.getUpperBound();
+    if (TypesUtils.isBoxedPrimitive(subtypeUpperBound.getUnderlyingType())
         && supertype instanceof AnnotatedPrimitiveType) {
-      upperBound = supertype.atypeFactory.getUnboxedType((AnnotatedDeclaredType) upperBound);
+      subtypeUpperBound =
+          supertype.atypeFactory.getUnboxedType((AnnotatedDeclaredType) subtypeUpperBound);
```

Comment:
It's surprising to see `supertype` used here when the assignment is all about subtypes.  Can you use `subtype` here instead of `superttype`?  If not, why?